app/views/teacher/new_student.erb

```diff
@@ -1 +1,30 @@
-<h1>Where teachers added a new student</h1>
+<h1>This is where a teaher adds new students.</h1>
+
+<% if @student.errors.any? %>
+  <div>
+    <strong>Some issues with your entry, please fix them:</strong>
+    <ul>
+      <% @student.errors.full_messages.each do |student| %>
```

Comment:
|student| should work
Though, for code legibility, it's better to rename to something like |error| or |message|